dev/devicelab/lib/tasks/sample_catalog_generator.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import '../framework/adb.dart';
+import '../framework/framework.dart';
+import '../framework/ios.dart';
+import '../framework/utils.dart';
+
+TaskFunction createSampleCatalogGenerator() {
+  return new SampleCatalogGenerator();
+}
+
+class SampleCatalogGenerator {
```

Comment:
FYI, this doesn't need to be a class. Can be a function. I other places they are classes for historic reasons.